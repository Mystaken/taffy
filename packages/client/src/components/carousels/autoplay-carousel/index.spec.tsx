import React from 'react';
import { AutoplayCarousel, AutoplayCarouselItem } from '.';
import { shallow } from 'enzyme';

describe('<AutoplayCarousel />', () => {
  it('has rendered correctly', () => {
    const tree = shallow(<AutoplayCarousel />);
    expect(tree).toMatchSnapshot();
  });
});

describe('<AutoplayCarouselItem />', () => {
  it('has rendered correctly', () => {
    const tree = shallow(<AutoplayCarouselItem image="" />);
    expect(tree).toMatchSnapshot();
  });
});
