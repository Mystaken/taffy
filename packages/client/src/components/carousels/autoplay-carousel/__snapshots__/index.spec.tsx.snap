// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<AutoplayCarousel /> has rendered correctly 1`] = `ShallowWrapper {}`;

exports[`<AutoplayCarouselItem /> has rendered correctly 1`] = `ShallowWrapper {}`;
